Common/Source/Globals.cpp

```diff
@@ -17,9 +17,47 @@
 
 #if (WINDOWSPC>0)
 #include <wingdi.h>
+
+#include <windows.h>
+#include <tchar.h>
+
+typedef BOOL (WINAPI *LPFN_ISWOW64PROCESS) (HANDLE, PBOOL);
+
+LPFN_ISWOW64PROCESS fnIsWow64Process;
+
 #endif
 
 
+
+
+
+BOOL IsWow64()
+{
+#if (WINDOWSPC>0)
+  if(!UTF8PICTORIALS) return false;
+
+  BOOL bIsWow64 = FALSE;
+
+  //IsWow64Process is not available on all supported versions of Windows.
+  //Use GetModuleHandle to get a handle to the DLL that contains the function
+  //and GetProcAddress to get a pointer to the function if available.
+
+  fnIsWow64Process = (LPFN_ISWOW64PROCESS) GetProcAddress(
+      GetModuleHandle(TEXT("kernel32")),"IsWow64Process");
+
+  if(NULL != fnIsWow64Process)
+  {
+      if (!fnIsWow64Process(GetCurrentProcess(),&bIsWow64))
+      {
+          //handle error
+      }
+  }
+  return bIsWow64;
+#else
+  return UTF8PICTORIALS;
+#endif
+}
+
```

Comment:
don't work for PCX64 target

Reply:
with IsWow64Process, you check if process is 32bit windows applications runing on 64bit windows, not a 64bit process running on 64bit windows

Reply:
OK, what do you recomend to do? 
(Keep in mind, it is configurable anyway. Default is the Bitmaps not UTF8)